pkg/scheduler/framework/plugins/noderesources/fit_test.go

```diff
@@ -1095,7 +1095,7 @@ func TestEventsToRegister(t *testing.T) {
 			"Register events with InPlacePodVerticalScaling feature enabled",
 			true,
 			[]framework.ClusterEventWithHint{
-				{Event: framework.ClusterEvent{Resource: "Pod", ActionType: framework.Update | framework.Delete}},
+				{Event: framework.ClusterEvent{Resource: "Pod", ActionType: framework.UpdatePodRequest | framework.Delete}},
```

Comment:
That is a good point. So, you mean we can create `PodRequestScaledDown` instead of `UpdatePodRequest`.

The difference would be:
- `PodRequestScaledDown`: Filter out events that represent scaling up in `podSchedulingPropertiesChange`.
- `UpdatePodRequest`: Filter out such events in QHint of resource fit plugin instead.

Looks similar, but `PodRequestScaledDown` has a certain advantage actually; it filters out events before putting events in inFlightEvents. So, it could help reducing the memory consumed by inFilghtEvents for sure. (https://github.com/kubernetes/kubernetes/issues/120622)

@alculquicondor What do you think; `UpdatePodRequest` vs `PodRequestScaledDown`. Any particular concern for `PodRequestScaledDown`?

Reply:
Yeah, I prefer we have PodRequestScaledDown, as scale ups won't make anything schedulable.